docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2823,6 +2827,17 @@ const participantsList = document.querySelector('#participant-list');
 const participantTable = document.querySelector('#participant-table');
 const participantButtons = document.querySelector('#participant-btn');
 
+participantTable.addEventListener('click', (event) => {
+  if (event.target.type === 'radio') {
+    const selectedParticipant = meeting.members.membersCollection.get(event.target.id);
+    if (!selectedParticipant) {
+      return;
+    }
+    const muteButton =  document.querySelector('#participant-btn .btn-group button:first-child');
```

Comment:
Can't we add ID to the button and do getElementByID? This will break if anyone adds another button before mute

Reply:
Changed to use getEelementById.